pom.xml

```diff
@@ -1094,6 +1094,23 @@
                                     </resources>
                                 </configuration>
                             </execution>
+                            <execution>
```

Comment:
No, every execution markup has assigned one directory, where the copied files are placed.